Bottom: 54b58c769ed3881babaf20863e5b2ceaf3ae3432
Top:    4d00360b7a912e4a06b97e6a747e8fd1bbb85958
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 17:06:44 -0500

Refresh of update-ptxdist-build-factories

---
diff --git a/roles/buildbot-worker-ptxdist/tasks/main.yml b/roles/buildbot-worker-ptxdist/tasks/main.yml
index 91dec6ce..b8ed812c 100644
--- a/roles/buildbot-worker-ptxdist/tasks/main.yml
+++ b/roles/buildbot-worker-ptxdist/tasks/main.yml
@@ -65,10 +65,6 @@
     dest: '{{ image_dir }}'
     force: yes
 
-- template:
-    src: pip.conf.j2
-    dest: '{{ image_dir }}/buildbot/.pip/pip.conf'
-
 - template:
     src: pydistutils.cfg.j2
     dest: '{{ image_dir }}/buildbot/.pydistutils.cfg'
